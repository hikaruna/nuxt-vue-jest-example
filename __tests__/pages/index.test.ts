import { shallowMount } from '@vue/test-utils'
import IndexPage from '~/pages/index.vue'

describe('index.vue', () => {
  test('renders "hello world"', () => {
    const wrapper = shallowMount(IndexPage)
    expect(wrapper.text()).toMatch("hello world")
  })
})
