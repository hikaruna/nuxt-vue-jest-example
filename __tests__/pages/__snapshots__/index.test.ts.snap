// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue renders "hello world" 1`] = `
<h1>
  hello world
</h1>
`;
